doc/api/http2.md

```diff
@@ -1797,9 +1797,28 @@ on the `Http2Server` after `msecs` milliseconds.
 
 The given callback is registered as a listener on the `'timeout'` event.
 
-In case of no callback function were assigned, a new `ERR_INVALID_CALLBACK`
+In case if `callback` is not a function, a new `ERR_INVALID_CALLBACK`
 error will be thrown.
 
+#### `server.timeout`
+<!-- YAML
+added: v8.4.0
+changes:
+  - version: v13.0.0
+    pr-url: https://github.com/nodejs/node/pull/27558
+    description: The default timeout changed from 120s to 0 (no timeout).
```

Comment:
nit:
```suggestion
    description: The default timeout changed from 120 seconds to 0 (no timeout).
```
here and below

Reply:
The same text (`120s`) is also used in `http.md` and will be used in `https.md` once #31692 lands. So, it's better to wait until that PR lands and change text in all places simultaneously.